README.md

```diff
@@ -6,19 +6,19 @@ To install the Python_Toolkit:
     1. [BHoM_Engine](https://github.com/BHoM/BHoM_Engine)  
     1. [BHoM_Adapter](https://github.com/BHoM/BHoM_Adapter)   
     1. [BHoM_UI](https://github.com/BHoM/BHoM_UI)  
-	  1. [Rhinoceros_Toolkit](https://github.com/BHoM/Rhinoceros_Toolkit)  
-	  1. [Grasshopper_Toolkit](https://github.com/BHoM/Grasshopper_Toolkit)  
-	  1. Python_Toolkit (this repo)
+    1. [Rhinoceros_Toolkit](https://github.com/BHoM/Rhinoceros_Toolkit) (Using Rhino and Grasshopper as an example of UI platform) 
```

Comment:
I don't understand this comment? What is it adding?

Reply:
My original thought is people may not choose Grasshopper as their UI platform (as we are saying below in a few lines), then compiling Rhino_Toolkit and Grasshopper_Toolkit are only one of the options for the UI platform. But reading it again now I'm not sure this is the best word here. Any suggestion?

Reply:
I would either add all of the UIs as options, but then needs updating whenever UIs change, or just remove this part. I would follow the example in some of the other ReadMe's for how the dependency on BHoM is handled to avoid having to copy it here - maybe just link back to BHoM will be enough.